common.gypi

```diff
@@ -36,7 +36,7 @@
 
     # Reset this number to 0 on major V8 upgrades.
     # Increment by one for each non-official patch applied to deps/v8.
-    'v8_embedder_string': '-node.14',
+    'v8_embedder_string': '-node.15',
```

Comment:
```suggestion
    'v8_embedder_string': '-node.16',
```
Since https://github.com/nodejs/node/commit/b460c5c5240f5c1cb34fea6eea088e882a2dd802 has just bumped this on master.